data/json/vehicleparts/frames.json

```diff
@@ -84,7 +97,20 @@
     "location": "structure",
     "color": "dark_gray",
     "durability": 1600,
-    "difficulty": 1,
+    "difficulty": 3,
+    "requirements": {
+      "install": {
+        "skills": [ [ "mechanics", 3 ] ],
+        "time": 240000,
+        "qualities": [ { "id": "GLARE", "level": 2 } ],
+        "tools": [ [ [ "welder", 200 ], [ "welder_crude", 300 ], [ "toolset", 300 ], [ "oxy_torch", 60 ] ] ],
```

Comment:
@mugling @Coolthulhu How should the oxy_tourch charges scale with the welder charges? I always see different ratios.